addon/src/lib/appStartup.js

```diff
@@ -18,10 +18,10 @@ export function startupDone() {
 
 export const startupObserver = {
   register() {
-    Services.obs.addObserver(this, 'sessionstore-windows-restored', false);
```

Comment:
May be a harmless change, worth omitting

Reply:
I fixed this, but git isn't marking it as outdated!

Reply:
prolly cus there is still a change on this line